src/test/fuzz/netbase_dns_lookup.cpp

```diff
@@ -29,8 +29,8 @@ FUZZ_TARGET(netbase_dns_lookup)
     };
 
     {
-        std::vector<CNetAddr> resolved_addresses;
-        if (LookupHost(name, resolved_addresses, max_results, allow_lookup, fuzzed_dns_lookup_function)) {
+        const std::vector<CNetAddr> resolved_addresses{LookupHost(name, max_results, allow_lookup, fuzzed_dns_lookup_function)};
+        if (!resolved_addresses.empty()) {
             for (const CNetAddr& resolved_address : resolved_addresses) {
                 assert(!resolved_address.IsInternal());
             }
```

Comment:
nit: Note that the empty-check is not needed anymore. If the vector is empty then iterating it will do nothing anyways:
```suggestion
        for (const CNetAddr& resolved_address : resolved_addresses) {
            assert(!resolved_address.IsInternal());
        }
```

Reply:
Nice, addressed!